resources/test/tests/idlharness-primary-int-obj.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>idlharness: Primary interface</title>
+  <script src="../../testharness.js"></script>
```

Comment:
Using relative pathing is preferable for some consumers because it allows running the tests over the `file:` protocol, but I can't argue with precedent. I may follow up with a separate patch to switch to relative pathing later.